treeherder/etl/common.py

```diff
@@ -52,14 +52,29 @@ def __getitem__(self, name):
         return value
 
 
-def get_remote_content(url, params=None):
-    """A thin layer of abstraction over requests. """
-    resp = requests.get(url,
-                        params=params,
-                        headers={'Accept': 'application/json'},
-                        timeout=settings.TREEHERDER_REQUESTS_TIMEOUT)
-    resp.raise_for_status()
-    return resp.json()
+def make_request(url, method='GET', headers={},
```

Comment:
I think using a dictionary as a default parameter is a bad idea here, as it could lead to unexpected behaviour.

See: https://pythonconquerstheuniverse.wordpress.com/category/python-gotchas/

I'd just set it to None by default and reinitialize it as empty if that's the case.